3rdparty/openssl/CMakeLists.txt

```diff
@@ -76,7 +78,10 @@ if (OE_SGX)
   # Special case where assembly file is generated by different script
   # (crypto/sha/asm/sha256-mb-x86_64.pl)
   list(APPEND OPENSSL_GENERATED_BYPRODUCTS crypto/sha/asm/sha256-x86_64)
-  list(APPEND OPENSSL_3_GENERATED_BYPRODUCTS crypto/sha/asm/sha256-x86_64)
+  # list(APPEND OPENSSL_3_GENERATED_BYPRODUCTS crypto/sha/asm/sha256-x86_64)
+
+  list(APPEND OPENSSL_3_GENERATED_BYPRODUCTS crypto/bn/asm/rsaz-2k-avx512
```

Comment:
Add a comment explaining why these files are appended seperately. (E.g. "Intel SSL repo doesn't include these assembly files, we need to generate them using OpenSSL .pl scripts as we do for OpenSSL 1.1.1")